src/leader.py

```diff
@@ -152,6 +155,41 @@ def lead(self):
             self.turn(code, position)
 
 
+    def drives_in_circle(self):
+        """
+        Drives in a circle.
+        """
+        self.turn_left()
+        self.bw.speed(MAX_SPEED)
+        self.bw.forward()
+
+
+    def drive_wide_winding(self):
+        """
```

Comment:
this can be on one line
```"""docstring"""```

Reply:
I think we do this in other classes. Double check before doing it.

Reply:
It is done that way in a few simple functions in the follower, will change in a sec.